migration_scripts/1.10/bitTable_remover

```diff
@@ -0,0 +1,31 @@
+#! /bin/bash
```

Comment:
All of our migration scripts are strictly Python. I'm not opposed to this being here, but then the readme below needs to very clearly show this migration script needs to be run differently. Regardless, I personally don't think this even warrants a migration script, it's not some sort of large-scale data transfer. If anything, I think this may set a dangerous precedent for people expecting a migration script for every minute refactor we do, which can easily become dangerous because not every refactor can be fixed by a simple search and replace

Reply:
Agreed, changes in this PR are minimal compared to bigger data migrations like trainers, and instructions for updating custom code can be added in the changelog.

Reply:
Regarding script language, it uses only things that are already present on all supported build environments by definition, should probably be `#! /bin/sh` though.
If can keep the scripts in my repo if people want it for working with pre-1.10 or non-Expansion versions.
I think both Tertu and I started thinking about using a script for this because it's easily "search & replaced".

Reply:
I think the fact it's trivial to search and replace it is exactly why we don't need a migration script, especially since I think we can count the amount of ``gBitTable`` references made in new code on one hand.

Reply:
Yea, I'll remove it and put it in my repo.